Sync/Synchronizers/Bookmarks/BookmarksRepairRequestor.swift

```diff
@@ -144,18 +145,26 @@ class BookmarksRepairRequestor {
 
         guard ids.count <= MaxRequestedIDs else {
             log.info("Not starting a repair as there are over \(MaxRequestedIDs) problems")
-            // TODO: TELEMETRY
-            // let extra = { flowID, reason: `too many problems: ${ids.size}` }
-            // this.service.recordTelemetryEvent("repair", "aborted", undefined, extra)
+            let extra = [
+                "flowID": flowID,
+                "reason": "too many problems: \(ids.count)"
+            ]
+
+            let event = Event(category: "sync", method: "repair", object: "aborted", extra: extra)
+            recordTelemetry(event: event)
```

Comment:
Sure! The only reason I need to use `self` in the places I did is because when calling a method from within a closure you need to use `self`.